routes/users.js

```diff
@@ -7,11 +10,28 @@ router.route('/').get((req, res) => {
     .catch(err => res.status(400).json('Error: ' + err));
 });
 
+
+
+const hashPassword = (pwd) => {
```

Comment:
Im thinking it might make sense to make a helper.js file for each router/global scope and put functions like this in their respective file. That was we can import functions like this in multiple places if we need them again. 

For this function we might only need it in this once spot but because we are getting into helper function with this I figured it would be time to consider starting this flow.